src/ws/main.c

```diff
@@ -53,6 +54,7 @@ static gboolean  opt_version      = FALSE;
 static GOptionEntry cmd_entries[] = {
   {"port", 'p', 0, G_OPTION_ARG_INT, &opt_port, "Local port to bind to (9090 if unset)", NULL},
   {"address", 'a', 0, G_OPTION_ARG_STRING, &opt_address, "Address to bind to (binds on all addresses if unset)", "ADDRESS"},
+  {"unix", 'U', 0, G_OPTION_ARG_STRING, &opt_unix, "Listen on this unix address", "PATH"},
```

Comment:
Also needs to go into doc/man/cockpit-ws.xml . It's a tiny bit sad to see features being added to ws, while we wanted to get rid of it, but it's also nice to clean up the TCP parts.

Reply:
What is that for, OOI? the current patch series does not seem to actually use it at all, the "get socketpair connection" seems much more elegant anyway?

Reply:
This is eventually meant to enable cockpit-ws on a unix socket inside of the container in order to enable it to be used with cockpit-tls.  We could defer this patch until we actually deal with #16650 if you prefer.

Reply:
It looks fine, I was just curious. As it's currently completely untested, I slightly prefer postponing it. But if it's in any way useful for you already, then let's just add it to the manpage and land it.

Reply:
I dropped it for now.  Let's handle it in #16650.